sql/core/src/main/scala/org/apache/spark/sql/catalyst/analysis/ResolveSessionCatalog.scala

```diff
@@ -545,6 +538,23 @@ class ResolveSessionCatalog(
         replace,
         viewType)
 
+    case ShowViews(resolved: ResolvedNamespace, pattern) =>
+      resolved match {
+        case SessionCatalogAndNamespace(_, ns) =>
+          // Fallback to v1 ShowViewsCommand since there is no view API in v2 catalog
+          val namespace = if (ns.isEmpty) {
```

Comment:
can `ns` be empty? We always fill it with current database if it's empty, right?

For example, in this file we have
```
case ShowTables(SessionCatalogAndNamespace(_, ns), pattern) =>
      assert(ns.nonEmpty)
```

Reply:
Yea, that's right. Updated in 2f8074a48c23f3c922958fa95c56cb5ffd64bc35 to get along with `ShowTables` code style, thanks!